src/main/java/ch/aaap/harvestclient/core/Harvest.java

```diff
@@ -139,6 +139,16 @@ public Harvest(Config config) {
         this.authToken = config.getString("harvest.auth.token");
         this.accountId = config.getString("harvest.auth.accountId");
 
+        String authTokenEnv = System.getenv("HARVEST_AUTH");
```

Comment:
this kind of logic belongs to the caller of this constructor, otherwise we have would have to introduce an ENV variable for every configuration option. 
Since we are a library I prefer the config to be explicit, the client application using this library can do:


```
Config config = ConfigFactory.defaultReference();
     config = config.withValue("harvest.auth.token", ConfigValueFactory.fromAnyRef("YOUR_TOKEN"));
     config = config.withValue("harvest.auth.accountId", ConfigValueFactory.fromAnyRef("YOUR_ACCOUNT_ID"));
      harvest = new Harvest(config)
```
and load YOUR_TOKEN and YOUR_ACCOUNT_ID from the environment